src/main/java/net/mcreator/preferences/PreferencesData.java

```diff
@@ -138,7 +138,7 @@
 	}
 
 	public enum WorkspaceIconSize {
-		LARGE, SMALL, LIST
+		LARGE, MEDIUM, SMALL, LIST, TILES, DETAILS
```

Comment:
Keep inner name LARGE for tiles so existing workspaces keep tiles by default.

Reply:
You mean, In the code you keep this view's name LARGE, but still naming it tiles as UI String:
![image](https://user-images.githubusercontent.com/60476871/118363512-c4bec500-b5a5-11eb-98f7-9b4fddc7a5ba.png)

Reply:
Yes, for backwards compatibility. Or alternatively rename parameter in preferencesdata so it resets and set new default value to TILE there